### PR TITLE
Added new R libraries

### PR DESCRIPTION
**Pull request by sknigh**

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-mixtools/package.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+##############################################################################
+from spack import *
+
+
+class RMixtools(Package):
+    """
+    A collection of R functions for analyzing finite mixture models.
+    """
+
+    homepage = "http://www.example.co://cran.r-project.org/web/packages/mixtools/index.html"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-mixtools/package.py]: I think this URL got messed up

File: var/spack/repos/builtin/packages/r-network/package.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+class RNetwork(Package):
+    """
+    Tools to create and modify network objects. The network class can 
+    represent a range of relational data types, and supports arbitrary 
+    vertex/edge/graph attributes.
+    """
+
+    homepage = "http://www.example.com"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-network/package.py]: Homepage

File: var/spack/repos/builtin/packages/r-robustbase/package.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+class RRobustbase(Package):
+    """"
+    Essential" Robust Statistics. Tools allowing to analyze data with robust
+    methods. This includes regression methodology including model selections
+    and multivariate statistics where we strive to cover the book "Robust
+    Statistics, Theory and Methods" by 'Maronna, Martin and Yohai';
+    Wiley 2006.
+    """
+
+    homepage = "http://www.example.co://cran.r-project.org/web/packages/robustbase/index.html"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-robustbase/package.py]: Messed up URL

File: var/spack/repos/builtin/packages/r-coda/package.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+
+class RCoda(Package):
+    """
+    Provides functions for summarizing and plotting the output from Markov
+    Chain Monte Carlo (MCMC) simulations, as well as diagnostic tests of
+    convergence to the equilibrium distribution of the Markov chain.
+    """
+
+    homepage = "http://www.example.com"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-coda/package.py]: homepage

File: var/spack/repos/builtin/packages/r-latentnet/package.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+from spack import *
+
+
+class RLatentnet(Package):
+    """
+    Fit and simulate latent position and cluster models for statistical
+    networks.
+    """
+
+    homepage = ""
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-latentnet/package.py]: missing homepage

**Comment by sknigh** [comment]
Something's not right with this PR. Will try again later.

**Pull request closed without merging**